crates/testing/tests/tests_1/consensus_task.rs

```diff
@@ -101,17 +98,29 @@ async fn test_consensus_task() {
                 null_block::builder_fee(quorum_membership.total_nodes()).unwrap(),
             ),
         ],
-        outputs: vec![
+    ];
+
+    let expectations = vec![
+        Expectations::from_outputs(vec![all_predicates(vec![
+            exact(ViewChange(ViewNumber::new(1))),
+            quorum_proposal_validated(),
+            exact(QuorumVoteSend(votes[0].clone())),
+        ])]),
```

Comment:
This definitely looks a bit cumbersome. I wonder: is it easy to just add a macro like e.g. `all![exact(), quorum_proposal_validated(), exact()]` (though if it takes more than 2 minutes I probably wouldn't bother)

Reply:
Let me give 'er a try.

Reply:
Done.